scripts/test/integration-setup/index.js

```diff
@@ -0,0 +1,116 @@
+#!/usr/bin/env node
+
+'use strict';
+
+require('essentials');
+require('log-node')();
+
+const log = require('log').get('serverless:scripts');
```

Comment:
Let's use just `serverless` namespace (it's standalone script, so it's fine to just stick to `serverless`